build.gradle

```diff
@@ -22,7 +22,7 @@ sourceSets {
         resources { srcDir 'data' }
     }
     test {
-        java { srcDirs 'test'}
+        java { srcDirs 'test' }

```

Comment:
The tests also need the data folder at runtime, and I think need the maps folder as well. I'm curious how to old project folders to the runtime classpath. I'm working on adding a file that at least describes what you need to do in eclipse to get things to compile/test.